testutil/integration/example_test.go

```diff
@@ -104,6 +122,15 @@ func Example_oneModule() {
 	keys := storetypes.NewKVStoreKeys(authtypes.StoreKey)
 	authority := authtypes.NewModuleAddress("gov").String()
 
+	db := dbm.NewMemDB()
+	cms := store.NewCommitMultiStore(db, log.NewNopLogger(), metrics.NewNoOpMetrics())
```

Comment:
Maybe we should provide an helper for that, given it will be required in every integration test

Reply:
I've added it in `router.go`. LMK if it needs any changes